### PR TITLE
remove the startup shortcut from wix

### PR DESCRIPTION
**Pull request by xgess**
this was auto-starting all of the non-Gui services. when we changed the way auto-start works (so that it would actually work) this was still hanging around.

### CODE REVIEW

**Comment by taruti** [comment]
LGTM

File: go/install/install_windows.go [hunk 1]
```diff
@@ -261,9 +261,12 @@ func deprecatedStartupInfo(logFile *os.File) {
...
 		logFile.WriteString("Error getting AppDataDir\n")
 	} else {
-		if exists, err := libkb.FileExists(filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")); err == nil && exists == false {
+		autostartLinkPath := filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")
+		if exists, err := libkb.FileExists(autostartLinkPath); err == nil && exists == false {
 			logFile.WriteString("  -- Service startup shortcut missing! --\n\n")
-		} else if err != nil {
+			return
+		}
+		if err != nil {
```
Comment (zapu) [inline go/install/install_windows.go]: this is different then it used to be, is it expected? this is not the `err` from `libkb.FileExists`

Comment (xgess) [reply go/install/install_windows.go]: fixed

**Comment by taruti** [comment]
`deprecatedStartupInfo` is called from the log send codepath not on every startup.

File: go/install/install_windows.go [hunk 2]
```diff
@@ -374,7 +374,16 @@ func LsofMount(mountDir string, log Log) ([]CommonLsofResult, error) {
 	return nil, fmt.Errorf("Cannot use lsof on Windows.")
 }
 
+// delete this function and calls to it if present after 2022
+func deleteDeprecatedFileIfPresent() {
+	// this file is no longer how we do things, and if it's present (which it shouldn't be) it could
+	// cause unexpected behavior
+	autostartLinkPath := filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")
+	_ = os.Remove(autostartLinkPath)
```
Comment (taruti) [inline go/install/install_windows.go]: Just to make sure appDataDir ends in '\'?

File: go/install/install_windows.go [hunk 3]
```diff
@@ -374,7 +374,16 @@ func LsofMount(mountDir string, log Log) ([]CommonLsofResult, error) {
...
+}
+
 func GetAutostart(context Context) keybase1.OnLoginStartupStatus {
+	deleteDeprecatedFileIfPresent()
```
Comment (taruti) [inline go/install/install_windows.go]: This might be called more often than once per run, but that might be fine.

**Comment by taruti** [comment]
Also this fails to build:
```
..\install\install_windows.go:381:37: undefined: appDataDir
```

File: go/install/install_windows.go (hunk 3 above)
Comment (xgess) [reply go/install/install_windows.go]: yeah - agreed. not a huge deal.

File: go/install/install_windows.go (hunk 2 above)
Comment (xgess) [reply go/install/install_windows.go]: shouldn't matter `filepath.Join` looks smart enough to deal with it either way. 
and i made this code now matches almost exactly the existing code in `deprecatedStartupInfo`

**Comment by xgess** [comment]
> Also this fails to build:
> 
> ```
> ..\install\install_windows.go:381:37: undefined: appDataDir
> ```

yeah - fixed that :/

**Pull request merged**